### PR TITLE
Update hyprland workspaces

### PR DESCRIPTION
**Pull request by shobu13**
hyprland-workspaces introduce a new feature to add class to workspaces actually on screen instead of just currently focused workspace.

## Things done

Push hyprland workspaces from 2.0.3 to 2.0.4
add Sin Serhao as contributor

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix
Comment (SigmaSquadron) [inline maintainers/maintainer-list.nix:1]: Please *do not* format the `maintainers-list.nix` file. A treewide PR will soon do it for you, and formatting such a massive file without prior coordination may cause multiple unintended merge conflicts in the other branches of Nixpkgs.

**Review by SigmaSquadron — CHANGES_REQUESTED** [review]

File: maintainers/maintainer-list.nix
Comment (shobu13) [reply maintainers/maintainer-list.nix:1]: okay sorry, there is not much doc about formating of maintainer list, like for alphabetical sorting if there is a tool for

Comment (SigmaSquadron) [reply maintainers/maintainer-list.nix:1]: If you don't get the alphabetical sorting correct before pushing, the CI will tell you the maintainers-list file isn't sorted and will tell you exactly where to put your entry.

Also, please avoid marking comments as resolved if they weren't resolved yet.

File: pkgs/by-name/hy/hyprland-workspaces/package.nix
Comment (SigmaSquadron) [inline pkgs/by-name/hy/hyprland-workspaces/package.nix:1]: Please format package files in a separate commit.

**Comment by donovanglover** [comment]
Thanks for contributing! This package was already updated by r-ryantm here: https://github.com/NixOS/nixpkgs/pull/363119

You are still free to add yourself as a maintainer if you want the ability to merge PRs for `hyprland-workspaces` from r-ryantm.

**Comment by shobu13** [comment]
thanks for your time, sorry for the bother.

**Pull request closed without merging**